clients/githubrepo/graphql.go

```diff
@@ -224,8 +242,12 @@ func commitsFrom(data *graphqlData, repoOwner, repoName string) ([]clients.Commi
 				})
 			}
 			for _, review := range pr.Reviews.Nodes {
+				author := clients.User{
+					Login: string(review.Author.Login),
+				}
 				associatedPR.Reviews = append(associatedPR.Reviews, clients.Review{
-					State: string(review.State),
+					State:  string(review.State),
+					Author: &author,
```

Comment:
Nit: remove the `author` var and create the struct here directly.

Reply:
author is a pointer, I don't think you can assign a temporary pointer directly. If the compiler does not complain, I'll do that

Reply:
And I was wrong, below seem to work just fine! 
```
Author: &clients.User{
  Login: string(review.Author.Login),
},
```